internal/store/hpa.go

```diff
@@ -143,6 +143,56 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_hpa_status_average_value",
+			Type: metric.Gauge,
+			Help: "Average metric value observed by the autoscaler.",
```

Comment:
Are these metrics not already exposed by k8s?

Reply:
You can view them by querying metrics server or by viewing the hpa object. However, these metrics are not exported anywhere that is consumable by an observability stack.